test/dogma/config_test.exs

```diff
@@ -23,7 +23,8 @@ defmodule Dogma.ConfigTest do
     TemporaryEnv.delete( :dogma, :rule_set ) do
       TemporaryEnv.delete( :dogma, :override ) do
 
-        assert Dogma.RuleSet.All.rules == Config.build.rules
+        assert Enum.sort(Dogma.RuleSet.All.rules) ==
+          Enum.sort(Config.build.rules)
```

Comment:
The order of the rules returned by `Config.build.rules` changed when a 33rd rule was added because `Config.get_rules` builds a list of rules by pushing them into a map, and then calling `Dict.values`. `Dict.values` doesn't guarantee the order of the values returned. My guess is that maps have different internal representations of their values when growing from 32 to 33 key/value pairs which changes the ordering.